server/src/test/java/org/elasticsearch/indices/IndicesRequestCacheTests.java

```diff
@@ -629,7 +631,11 @@ public boolean isFragment() {
         }
     }
 
-    private class TestEntity extends AbstractIndexShardCacheEntity {
+    private static MappingLookup createMappingLookup() {
+        return new MappingLookup(Mapping.EMPTY, emptyList(), emptyList(), emptyList(), null, null, null);
+    }
```

Comment:
I think you can use `MappingLookup.EMPTY` here?

Reply:
almost :) in some places I do need a different instance, but I no longer need the method I think.